### DISCUSSION TITLE
How to replace the image on the main dashboard?

### DISCUSSION THREAD

**Discussion by Dennis-Zhang-SH** [ayamir/nvimdots#975] [General]
as title

**Comment by Dennis-Zhang-SH** [comment]
![image](https://github.com/ayamir/nvimdots/assets/51769674/8214fabe-94cf-4da4-8a85-c0b2daf2a199)

**Comment by charliie-dev** [comment]
https://github.com/ayamir/nvimdots/wiki/Issues#change-dashboard-startup-image

some ppl just dont read the wiki, right?
